codegen/src/test/java/org/web3j/codegen/unit/gen/java/MethodParserTest.java

```diff
@@ -27,7 +27,8 @@ public void testThatDeployMethodWasGenerated() {
         Optional<Method> deployMethod =
                 filteredMethods.stream().filter(m -> m.getName().equals("deploy")).findAny();
         MethodSpec deployMethodSpec =
```

Comment:
add in this class a test for a smart contract that contains 2 methods with the same name...even erc721

Reply:
ERC721 is hard to add in here, I have added duplicate functions in greeter contract, and added the test